src/select/parts/styles.scss

```diff
@@ -135,7 +135,7 @@ $inlineLabel-border-radius: 2px;
   // Stepped gradient background for inline label overlap between input and background.
   background-image: linear-gradient(
     to bottom,
-    transparent calc(100% - (awsui.$border-field-width + awsui.$border-control-focus-ring-shadow-spread + 5px)),
+    transparent calc(100% - (awsui.$border-width-field + awsui.$border-control-focus-ring-shadow-spread + 5px)),
```

Comment:
It looks like this brakes in [collection select filters](https://d21d5uik3ws71m.cloudfront.net/components/ac6b34785a9febf09da61c39f79b6166c6f38235/dev-pages/index.html#/light/app-layout/with-table-collection-select-filter), if you change the field width. It strikes through the input label.

Reply:
This needs more investigation I'll make a note to follow up.